shepherd_bms/include/datastructs.h

```diff
@@ -1,5 +1,66 @@
 #ifndef DATASTRUCTS_H
 #define DATASTRUCTS_H
 
+#include <nerduino.h>
+
+
+
+/*Individual chip data*/
+typedef struct ChipData_t
+{
+    uint16_t VoltageReading[12];          //store voltage readings from each chip
```

Comment:
Are we going to do open cell voltages? If so should they be in here, or a separate, less-frequently updated one?

Reply:
How would we get open cell voltages?

Reply:
Open cell voltage is the voltage when there is zero current flow and the voltage has stabilized. It's semi-relevant since it is the actual indicator of SOC, whereas the measured voltage will sag significantly while actually driving/drawing current.